soweego/linker/classify.py

```diff
@@ -94,14 +96,18 @@ def execute(catalog, entity, model, name_rule, threshold, dir_io):
 
         _add_missing_feature_columns(classifier, feature_vectors)
 
-        predictions = classifier.predict(feature_vectors) if isinstance(classifier, rl.SVMClassifier) else classifier.prob(feature_vectors)
+        predictions = classifier.predict(feature_vectors) if isinstance(classifier,
+                                                                        rl.SVMClassifier) else classifier.prob(
+            feature_vectors)
 
         # See https://stackoverflow.com/a/18317089/10719765
         if name_rule:
             LOGGER.info('Applying full names rule ...')
             predictions = DataFrame(predictions).apply(
                 _zero_when_different_names, axis=1, args=(wd_chunk, target_chunk))
 
+        predictions = DataFrame(predictions).apply(_one_when_wikidata_link_correct, axis=1, args=(target_chunk,))
```

Comment:
I think this `apply` should be fired **optionally**, in the same way as `name_rule`.